pkg/init/Dockerfile

```diff
@@ -28,8 +31,9 @@ ENTRYPOINT []
 CMD []
 WORKDIR /
 COPY --from=build /go/bin/init /
+COPY --from=build /tmp/bin /bin/
 COPY --from=build /go/bin/service /usr/bin/
 COPY --from=build usermode-helper /sbin/
 COPY --from=mirror /out/ /
 COPY etc etc/
-COPY bin bin/
+#COPY bin bin/
```

Comment:
You can nuke this I think

Reply:
Ah yes I think that is left over from testing